assets/js/app.js

```diff
@@ -1,25 +1,34 @@
-$('.js-toggle-menu').click(function() {
-  $('.l-header').toggleClass('is-collapse');
+$(".js-toggle-menu").click(function() {
+  $(".l-header").toggleClass("is-collapse");
```

Comment:
No need for jQuery here: https://dommagnifi.co/2016-05-16-basic-class-toggle-with-vanilla-js/